tools/CoqMakefile.in

```diff
@@ -189,7 +187,12 @@ else
 CAMLP4EXTEND=
 endif
 
+ifneq (, $(shell which "$(OCAMLFIND)" 2> /dev/null))
```

Comment:
Does this work, or does the leading space mess things up? (What does make do in `ifneq (, )`?)

Reply:
Good question, let's remove the space!

Reply:
Btw, if you could test the patch on the setup where the message was showing up, and confirm it is gone, it would be ideal.